src/config/dynamic-theme-fixes.config

```diff
@@ -4448,6 +4448,14 @@ CSS
 
 ================================
 
+downdetector.*
+xn--allestrungen-9ib.de
+
+INVERT
+img[class="img-fluid lazy"]
```

Comment:
This targets all images, I would prefer to not have inverted logo's. Some will be broken, others will be fixed. I think it's better to leave them alone.

Reply:
OK